### ISSUE TITLE
Question about evaluation and pretrained folder website

### ISSUE THREAD

**Issue by GqqGGqqG** [AlonLahav/MeshWalker#15]
Hi, hope you are well.
It's an inspiring work,  and I have some questions about the evaluation and test code.

1. The paper mentioned that several wallks were taken to predict the cls/seg result. But as the wallks may be different, the result u get after running evaluation_cls.py may be different. For example, I run the trainng script to train the model on cubes/human_seg dataset and run the evaluation code several times which give me different accuracy, some of them are higher than the reported num. So, how did you get the final reported accuracy. Run the evaluation code only once or run many times to get the average or take the minimum.
2.  I find out that the [code in this repo](https://github.com/AlonLahav/MeshWalker/blob/f3b73b1e165ec6598dc10826245df8d9b6c4d8eb/params_setting.py#L105) shows that the features for modelnet40 dataset is xyz rather than dxdydz, so which kind of features is used for the final modelnet40 cls result in the paper.
3. The pretrained folder website is not avaliable(403 Forbidden at 2021/12/12)(That's why I have the problem with Q2), could you upload it to google drive or dropbox or sth else?

Thanks for your work and the attention.

**Comment by AlonLahav** [comment]
Hi,
I'm glad my work helps you.
1. As I remember, the variance is not so high. I think I ran several times and took the average.
2. In the paper we've used xyz, which were slightly better than dxdydz.
3. Yes. I've checked the pretrained folder and you are right. will take care of it and update the README when a new location is set.
Thanks,
Alon.

**Comment by GqqGGqqG** [comment]
Thanks for you reply. Hope the new location can be placed sooner.
I have a another question is about how to get the final model, use the last model accroding to the [iters to train](https://github.com/AlonLahav/MeshWalker/blob/f3b73b1e165ec6598dc10826245df8d9b6c4d8eb/params_setting.py#L180)  or take the best  accroding to reported accuracy in "[log.txt](https://github.com/AlonLahav/MeshWalker/blob/f3b73b1e165ec6598dc10826245df8d9b6c4d8eb/utils.py#L123)"?

**Comment by AlonLahav** [comment]
The links are Fixed. 
Sorry for the delay.
Regarding the model to take, I've used the last one.
